### PR TITLE
Fix broken tests of fix fakeredis count system

### PR DESCRIPTION
**Pull request by senhalil**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,11 +9,11 @@ gem 'rack-cors'
 gem 'rake'
 gem 'thin'
 
-gem 'grape', '>=1.5.0' # Important fixes are introduced in v1.5.0 (see PR #2096 & #2098)
+gem 'grape', '>=1.5.1' # Important fixes are introduced in v1.5.1 (see PRs PR #2128, #2096 & #2098)
```
Comment (braktar) [inline Gemfile:12]: I have checked, the fix comes from PR #2103 (https://github.com/ruby-grape/grape/pull/2103/files)

**Review by braktar — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -9,11 +9,11 @@ gem 'rack-cors'
...
 gem 'grape-entity'
-gem 'grape_logging'
 gem 'grape-swagger'
 gem 'grape-swagger-entity'
+gem 'grape_logging'
```
Comment (fonsecadeline) [inline Gemfile:16]: Just so I understand, why did you change order ?

**Review by fonsecadeline — APPROVED** [review]

File: Gemfile (hunk 2 above)
Comment (braktar) [reply Gemfile:16]: `-` is before `_` in the 'unicode alphabetical order'

**Review by braktar — APPROVED** [review]

File: Gemfile (hunk 2 above)
Comment (senhalil) [reply Gemfile:16]: in rubocop we trust :)
